src/appkit.cs

```diff
@@ -18378,12 +18379,14 @@ partial interface NSWindow : NSAnimatablePropertyContainer, NSUserInterfaceItemI
 	
 		[Export ("contentRectForFrameRect:")]
 		CGRect ContentRectFor (CGRect frameRect);
-	
+
+		[Internal]
 		[Export ("initWithContentRect:styleMask:backing:defer:")]
-		IntPtr Constructor (CGRect contentRect, NSWindowStyle aStyle, NSBackingStore bufferingType, bool deferCreation);
+		IntPtr _Init (CGRect contentRect, NSWindowStyle aStyle, NSBackingStore bufferingType, bool deferCreation);
```

Comment:
Did you try a `[PostSnippet]` to avoid copying _normally_ generated code into manual bindings ?